### PR TITLE
Feature: Events

### PR DESCRIPTION
**Pull request by luftedar**
# Feature: Events

In this update:

- Added Calculator logic

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,42 +1,67 @@
...
         </div>
         <div id="button-area">
           <div className="row row-1">
-            <button type="button" className="calc-button first-line">AC</button>
-            <button type="button" className="calc-button first-line">+/-</button>
-            <button type="button" className="calc-button first-line">%</button>
-            <button type="button" className="orange-button calc-button first-line">/</button>
+            <button onClick={this.handleClick} type="button" className="calc-button first-line">AC</button>
+            <button onClick={this.handleClick} type="button" className="calc-button first-line">+/-</button>
+            <button onClick={this.handleClick} type="button" className="calc-button first-line">%</button>
```
Comment (safafa) [inline src/components/Calculator.js:35]: - [x] When I try any number modulo zero the app crash. Kindly fix the issue maybe try displaying a message like when the user tries dividing by zero.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @luftedar , You did a great job so far 👏🏼 👏🏼 

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by luftedar** [comment]
### Applied Required Change:
- https://github.com/luftedar/math-magician/pull/3#discussion_r757026067

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**